web/src/main/java/org/openmrs/module/web/ModuleResourcesServlet.java

```diff
@@ -61,7 +61,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 		
 		response.setDateHeader("Last-Modified", f.lastModified());
 		response.setContentLength(Long.valueOf(f.length()).intValue());
-		String mimeType = getServletContext().getMimeType(f.getName());
+		String mimeType = OpenmrsUtil.getFileMimeType(f);
```

Comment:
Let's actually undo this change as well, since it shouldn't be necessary with the change to web.xml.

Reply:
@ibacher  I have fixed it. Thanks so much.